yt-dlp-py.py

```diff
@@ -1,53 +1,65 @@
 import subprocess
 from pprint import pprint
 import inquirer
-typeQuestion = [
-    inquirer.List(
-        "type",
-        message="What type of import are you using?",
-        choices=["batch", "link"],
-    ),
-]
-typeState = inquirer.prompt(typeQuestion)
+from classes import FileFormat, ImportType, UploadInformation
+
+#region Helper Functions
```

Comment:
Below are several helper functions. Not only do functions prevent similar code from being repeated it also aids in readability. The function name services as it's own comment. Allowing to reader to understand what is happening inside the function at a glance.